public/app/features/plugins/extensions/logs/log.ts

```diff
@@ -32,7 +32,6 @@ export class ExtensionsLog {
   }
 
   warning(message: string, labels?: Labels): void {
-    console.warn(message, { ...this.baseLabels, ...labels });
```

Comment:
Hmm, I am wandering if we should only remove this console warning in production (maybe there are things in there that devs could take action on during development?)

Reply:
Yes good shout. Will fix!